distribution/lib/Standard/Base/0.0.0-dev/src/Data/List.enso

```diff
@@ -120,7 +120,7 @@ type List
        - f: The binary function used to combine elements of the list.
 
        In general, the result of
-           (Cons l0 <| Cons l1 <| ... <| Cons ln) . fold init f
+           (Cons l0 &lt;| Cons l1 &lt;| ... &lt;| Cons ln) . fold init f
```

Comment:
What makes this change needed?

It looks bad for text users, and it feels like it should be possible to allow these symbols at least in the code sections (ones indented with 4 spaces).

Reply:
It's needed to render in the Component Browser correctly and the Docusaurus site.

Reply:
Maybe we should move this piece of code into an example block then?

Reply:
Yes happy to.